proto/otaclient_iot_logging_server_v1.proto

```diff
@@ -34,18 +35,23 @@ enum LogLevel {
   ERROR = 5;
   FATAL = 6;
 }
+
 enum ErrorCode {
-  OK = 0;
-  FAILED = 1;
+  NO_FAILURE = 0;         // Success
+  SERVER_QUEUE_FULL = 1;  // Error: Server queue is full
+  NOT_ALLOWED_ECU_ID = 2; // Error: Specified ECU ID is not allowed
+  NO_MESSAGE = 3;         // Error: No message in the request
 }
+
 message PutLogRequest {
-  LogType type = 1;
-  uint64 timestamp = 2;
-  LogLevel level = 3;
-  string data = 4;
+  string ecu_id = 1;    // target ECU ID
```

Comment:
`ecu_id` is necessary as input because it use to set log stream.

Reply:
Yes, with `ecu_id`, we can maintain the queue for each ECU, and ensure the fairness of log uploading, preventing one of the ECU flooding the otaclient-iot-logger to make other ECU hard to upload logs.